### PR TITLE
API page styling updates

### PR DESCRIPTION
**Pull request by epwinchell**
Old
Screenshot 2023-02-22 at 10 02 31 AM

New
Screenshot 2023-02-22 at 9 55 49 AM

### CODE REVIEW

File: src/components/APIDoc/ApiDoc.tsx [hunk 1]
```diff
@@ -43,23 +43,34 @@ export const ApiDoc: React.FunctionComponent<ApiDocProps> = props => {
...
-                <SecuritySchemeList schemes={Object.values(openapi.components.securitySchemes).map(s => deRef(s, openapi))} />
                 <Divider
-                    className="apid-c-divider pf-u-pt-xl"
+                    className="apid-c-divider pf-u-pb-md"
                     inset={{default: 'insetNone',}}
                 />
+                <TextContent className="pf-u-pb-lg">
+                    <Text component={TextVariants.h2}>
+                      Principals
+                    </Text>
```
Comment (tahmidefaz) [inline src/components/APIDoc/ApiDoc.tsx:68]: I think, this part shouldn't be hard coded. This information needs to be inferred from the API docs.

**Review by tahmidefaz — CHANGES_REQUESTED** [review]
I just had 1 comment. That section will be different for different APIs.
We can circle back on it later also. It might need some thinking. :thinking:

File: src/components/APIDoc/ApiDoc.tsx (hunk 1 above)
Comment (josejulio) [reply src/components/APIDoc/ApiDoc.tsx:68]: Yes, the principals is a tag or section from a particular API

**Review by josejulio — APPROVED** [review]

**Review by tahmidefaz — APPROVED** [review]
Lgtm!

**Pull request merged**